### DISCUSSION TITLE
Dịch support

### DISCUSSION THREAD

**Discussion by minhduc0711** [mlbvn/handson-ml2-vn#28] [Terminology] [answered]
Ngữ cảnh:
> This assumes that all labels are equally important, however, which may not be the case. In particular, if you have many more pictures of Alice than of Bob or Charlie, you may want to give more weight to the classifier's score on pictures of Alice.
One simple option is to give each label a weight equal to its __support__ (i.e., the number of instances with that target label).

Mình thấy từ này không được dùng rộng rãi cho lắm, hình như chỉ được dùng trong scikit-learn
![image](https://user-images.githubusercontent.com/34294427/108451251-1272ac80-7299-11eb-8188-9150ead2717a.png)

Vì vậy nên mình đề xuất giữ nguyên từ này, vì nếu dịch là `hỗ trợ` thì cũng không gợi nghĩa lắm. Mọi người thấy sao nhỉ? @mlbvn/handson-ml2

**Comment by tiepvupsu** [comment, marked as answer]
Mình có thể bỏ không dịch từ `support` mà sử dụng luôn cụm trong ngoặc: "số phần tử của nhãn tương ứng".
